src/types/api/consumer.ts

```diff
@@ -0,0 +1,203 @@
+/* eslint-disable camelcase */
+
+// endpoint: /json/describe_consumer
+// source: https://github.com/ydb-platform/ydb/blob/main/ydb/public/api/protos/ydb_topic.proto
+
+// Original proto file doesn't specify optional fields, so every field is considered optional
+export interface DescribeConsumerResult {
+    self?: Entry;
+    consumer?: Consumer;
+    partitions?: PartitionInfo[];
+}
+
+// Partition info types differs for consumer and topic, although they are very similar
+export interface PartitionInfo {
+    /** int64  */
+    partition_id?: string;
+
+    // Is partition open for write.
```

Comment:
I suppose these annotations can as well be jsdoc, to appear in autocomplete popups

Reply:
I agree, this is a very good idea. I will do that